package.json

```diff
@@ -1,7 +1,8 @@
 {
```

Comment:
looks like you're using the prettier atom plugin ;)

would you exclude package.json from prettier?

imo, formatting package.json reliably makes it more confusing to parse

Reply:
Just checked, I have VSCode Prettier and package.json is included -- any ideas?

Reply:
I have the same problem. Simplest fix is to create a .prettierignore file and add package.json to it. You might even be able to point to a .prettierignore outside of your repo using VSCode Prettier's prettier.ignorePath option.